src/modules/common/components/Dropdown.js

```diff
@@ -0,0 +1,102 @@
+// @flow
+
+import React from 'react'
+import { Wrapper, Button, Menu, MenuItem } from 'react-aria-menubutton'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faAngleDown } from '../../app/constants/icons'
+import type { ThemeType } from '../../theme/constants/theme'
+import styled from 'styled-components'
+
+const StyledWrapper = styled(Wrapper)`
+  position: relative;
+  box-sizing: border-box;
+`
+
+const StyledMenuButton = styled(Button)`
+  display: flex;
+  cursor: pointer;
+  box-sizing: border-box;
+  align-items: center;
+  justify-content: space-between;
+  border-width: 1px;
+  border-style: solid;
+  border-color: ${props => props.theme.colors.textDecorationColor};
+  border-radius: 4px;
+`
+
+const StyledMenuButtonText = styled.div`
+  display: flex;
+  padding: 8px;
+  color: ${props => props.theme.colors.textSecondaryColor};
+`
+
+const StyledMenuButtonIcon = styled.div`
+  padding: 8px 16px;
+`
+
+const StyledMenu = styled(Menu)`
+  position: absolute;
+  top: 100%;
+  z-index: 1;
+  display: flex;
+  width: 100%;
+  box-sizing: border-box;
+  flex-direction: column;
+  margin: 8px 0;
+  border-radius: 4px;
+  background-color: ${props => props.theme.colors.backgroundColor};
+  box-shadow: 0 0 0 1px ${props => props.theme.colors.textDecorationColor},
+    0 4px 11px ${props => props.theme.colors.textDecorationColor};
+`
+
+const StyledMenuItem = styled(MenuItem)`
+  display: flex;
+  padding: 8px;
+  color: ${props => props.theme.colors.textSecondaryColor};
+  cursor: pointer;
+
+  :focus,
+  :hover {
+    background-color: ${props => props.theme.colors.textDecorationColor};
```

Comment:
Not exactly sure what color to use here.

Reply:
Black text color is fine, but the difference between #fafafa and white is nearly not visible here :/
![image](https://user-images.githubusercontent.com/29095487/80607680-12699d80-8a36-11ea-9730-3b9e21e48ea3.png)

Reply:
Maybe try #EDEDED. This should be the equivalent of the default color without an saturation, I think.

Reply:
@Taggotty we have some colors defined in our theme and should stick to them if possible. Therefore I would try not to introduce a new color here. What do you think about the color in the screenshot here https://user-images.githubusercontent.com/29095487/80475778-b0336e80-8949-11ea-8417-2fc35eaffbe0.png

Reply:
Is the font black? It looks kind of lighter than the one above. I think textDecorationColor and black text should be fine.

Reply:
Anyway I think the color is fine. It just doesn't really seem to pop.

Reply:
Now it should be black, yes. The screenshot below was before the change of the text color.